zcash_client_sqlite/src/testing.rs

```diff
@@ -465,6 +468,7 @@ impl<Cache> TestState<Cache> {
             ovk_policy,
             min_confirmations,
             change_memo,
+            ShieldedProtocol::Sapling,
```

Comment:
Make this an argument of the test helper method (it is intended to only provide the common internal parameters).

Reply:
This will change when we start testing Orchard functionality, because I want to refactor the tests to make our existing Sapling pool tests reusable for Orchard; making this change right now is more invasive than necessary.

Reply:
In that case, open an issue for this.